### PR TITLE
Removed duplicate line items and ordered the results by date

### PR DESCRIPTION
**Pull request by gowsik-ragunath**
## Summary

- Removed the duplicate line item entries in the `New line item` dropdown and `Add multiple line item` modal.
- Ordered record by `work_date`.

## Preview

https://www.loom.com/share/cd22f158de294573991b963b03810c46

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update

### Checklist:

- [X] I have manually tested all workflows
- [X] I have performed a self-review of my own code
- [ ] I have added automated tests for my code

### CODE REVIEW

File: app/models/client.rb [hunk 1]
```diff
@@ -59,7 +59,7 @@ def new_line_item_entries(selected_entries)
          timesheet_entries.note as description,
          project_members.hourly_rate as rate,
          timesheet_entries.duration as qty"
-      ).where.not(id: selected_entries)
```
[the remark this answers is not in the document]
Comment (gowsik-ragunath) [reply app/models/client.rb:62]: Yeah @akhilgkrishnan  - updated the existing test cases for this method.

**Review by akhilgkrishnan — APPROVED** [review]
LGTM

**Comment by akhilgkrishnan** [comment]
@mohinid can you check why the CI is failing. Cypress packages are not installing

**Comment by mohinid** [comment]
> @mohinid can you check why the CI is failing. Cypress packages are not installing

@apoorv1316 Can you please check.

**Comment by apoorv1316** [comment]
@gowsik-ragunath Why this PR is even installing cypress on CI I haven't merged the PR in which we are enabling the cypress test on CI. It's still open https://github.com/saeloun/miru-web/pull/490. cc @mohinid

**Comment by apoorv1316** [comment]
@gowsik-ragunath I am also not sure. Anway I re-run the job and all the checks seem fine now.

**Comment by supriya3105** [comment]
@apoorv1316 Please retest this bug and approve/reject accordingly.

**Review by rohitjoshixyz — COMMENTED** [review]
We should add an index on work date if we are sorting based on it cc @keshavbiswa

**Review by apoorv1316 — APPROVED** [review]
Thanks a lot, @gowsik-ragunath for fixing this. Really appreciate it. cc @supriya3105

**Comment by supriya3105** [comment]
@gowsik-ragunath Merging this. Added a new Notion card for @rohitjoshixyz 's feedback.

**Pull request merged**